### PR TITLE
Fix class components

### PR DESCRIPTION
**Pull request by stuttskl**
I changed the AlertBox from a class component to a functional component. I also added 'else ifs' to the alert generator.

### CODE REVIEW

File: src/Components/main.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ import Container from '@material-ui/core/Container';
 import Grid from '@material-ui/core/Grid';
 import Divider from '@material-ui/core/Divider';
 import "../styles/Main.css";
+import Alert from '@material-ui/lab/Alert';
```
Comment (tdraxler) [inline src/Components/main.js:18]: What's this for?

**Review by tdraxler — APPROVED** [review]

File: src/Components/main.js (hunk 1 above)
Comment (stuttskl) [reply src/Components/main.js:18]: That is a typo :D Looks like vscode pulled in 'Alert' when I was typing 'AlertBox'

Comment (tdraxler) [reply src/Components/main.js:18]: Ohhh ok :D
Also, weren't we going to have the alerts as a banner or something? (and red in color)

**Review by Cortesn — APPROVED** [review]
Looks great!

**Pull request merged**